### PR TITLE
#149798581 Configure API base url for contributions list

### PR DESCRIPTION
**Pull request by andela-uofoegbu**
#### What does this PR do?
Configure contributions list endpoint for the different environments
#### Description of Task to be completed?
The correct API_BASE_URL for each environment should be supplied for constructing the contributions list endpoint.
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#149798581](https://www.pivotaltracker.com/n/projects/2059149/stories/149798581)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
     ANDELA_API_BASE_URL: 'https://api.andela.com',
-    API_BASE_URL: '',
+    API_BASE_URL: 'https://private-3b35e-learningmapapi.apiary-mock.com',
     BASE_URL: 'http://learningmap-dev.andela.com:3000',
   },
-  production: {},
+  production: {
+    ANDELA_API_BASE_URL: 'https://api.andela.com',
+    API_BASE_URL: 'https://private-3b35e-learningmapapi.apiary-mock.com',
+    BASE_URL: 'http://learningmap-prod.andela.com:3000',
```
Comment (andela-kakpobome) [inline src/config/index.ts]: ### Critical

Why's this at port 3000 and not pointing to our Heroku prod build?

File: src/config/index.ts [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+  },
   staging: {
     ANDELA_API_BASE_URL: 'https://api.andela.com',
```
Comment (andela-kakpobome) [inline src/config/index.ts]: ### Critical

This should point to Andela's staging gateway i.e `api-staging.andela.com`

Comment (andela-uofoegbu) [inline src/config/index.ts]: Fixed this.

File: src/config/index.ts (hunk 1 above)
Comment (andela-uofoegbu) [inline src/config/index.ts]: I meant to ask about this. I'm not sure what our production base URL is, seeing as you told me to use similar values as in development

`learningmap.andela.com` ?

File: src/actions/ContributionsAction.ts [hunk 3]
```diff
@@ -1,4 +1,6 @@
+import axios from 'axios';
 import MockContributionApi from '../api/MockContributionApi';
+import {config} from '../config/index';
```
Comment (andela-kakpobome) [inline src/actions/ContributionsAction.ts]: Space out the braces

**Pull request merged**